website/docs/Ecosystem.md

```diff
@@ -19,3 +19,11 @@ MemGPT enables LLMs to manage their own memory and overcome limited context wind
 [Microsoft Fabric](https://learn.microsoft.com/en-us/fabric/get-started/microsoft-fabric-overview) is an all-in-one analytics solution for enterprises that covers everything from data movement to data science, Real-Time Analytics, and business intelligence. It offers a comprehensive suite of services, including data lake, data engineering, and data integration, all in one place. In this notenook, we give a simple example for using AutoGen in Microsoft Fabric.
 
 - [Microsoft Fabric + AutoGen Code Examples](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_microsoft_fabric.ipynb)
+
+## Ollama + AutoGen
+
+![Ollama Example](img/ecosystem-ollama.png)
```

Comment:
I love the graphic but I really want to be able to copy and paste this. Can you use just the image and then use a code block?

Reply:
I changed it to its official logo, making the style consistent with other integrations.